build/download-saturn-l2.js

```diff
@@ -54,25 +55,33 @@ async function main () {
 
         if (res.status >= 300) {
           throw new Error(
-            `Cannot fetch saturn-l2 binary ${name}: ${res.status}\n${await res.text()}`
+            `Cannot fetch saturn-l2 binary ${name}: ${res.status}\n` +
+            await res.text()
```

Comment:
We could also ignore template literals, check out the first commit for that version

Reply:
I prefer to add the `+` sign at the next line. That way adding a new line or removing the last line does not require changes in other lines.

I think it's also common to indent the lines after the first one by one level.

```js
            `Cannot fetch saturn-l2 binary ${name}: ${res.status}\n`
              + await res.text()
```

IIRC, Prettier would format this line like this - placing the line break inside the `${}` expression. I find the result a bit difficult to read, but otherwise it has many nice properties.

```js
            `Cannot fetch saturn-l2 binary ${name}: ${res.status}\n${
              await res.text()
            }`
```

Reply:
<img width="516" alt="Screenshot 2023-02-08 at 23 17 45" src="https://user-images.githubusercontent.com/10247/217663848-1a084d07-730c-42fe-bb56-746b79e73179.png">

This is currently prevented, let's tackle it in a separate PR

Reply:
> let's tackle it in a separate PR

👍🏻